app/controllers/concerns/foreman/controller/parameters/discovery_rule.rb

```diff
@@ -5,6 +5,8 @@ module Foreman::Controller::Parameters::DiscoveryRule
     def discovery_rule_params_filter
       Foreman::ParameterFilter.new(::DiscoveryRule).tap do |filter|
         filter.permit :name, :search, :hostname, :priority, :enabled, :hostgroup, :hostgroup_id, :max_count
+        filter.permit :organization_ids => []
```

Comment:
`include Foreman::Controller::Parameters::Taxonomix` and just call `add_taxonomix_params_filter(filter)` instead, as it permits other related parameters automatically.